pkgs/development/python-modules/omorfi/default.nix

```
@@ -0,0 +1,24 @@
+{ buildPythonPackage
+, pkgs
+, lib
+, omorfi
+, hfst
+}:
+
+buildPythonPackage {
+  inherit (pkgs.omorfi) src version;
+
+  pname = "omorfi";
+  doCheck = false;
+  propagatedBuildInputs = [
+    hfst-python
+  ];
+  sourceRoot = "source/src/python";
```

```suggestion
  pname = "omorfi";
  inherit (pkgs.omorfi) src;

  sourceRoot = "source/src/python";

  propagatedBuildInputs = [
    hfst-python
  ];

  doCheck = false;
```
normally we would order the attributes like this.

Also doCheck should get a comment when they are disabled.

The repo has tests so I am wondering why they are disabled https://github.com/flammie/omorfi/tree/main/test

---

The tests don't run due to some broken imports like this https://github.com/flammie/omorfi/blob/main/src/python/omorfi/entryguessing/plurale_tantum.py#L20

We could either keep the tests disabled or patch all of those faulty imports. The latter is probably better since those modules are exported in `omorfi-python` as well.